x/superfluid/keeper/stake.go

```diff
@@ -154,11 +152,6 @@ func (k Keeper) validateLockForSFDelegate(ctx sdk.Context, lock *lockuptypes.Per
 	}
 
 	denom := lock.Coins[0].Denom
-	// If the denom is a concentrated liquidity token, we just strip the position data from the denom
-	if strings.HasPrefix(denom, cltypes.ClTokenPrefix) {
-		index := strings.LastIndex(denom, "/")
-		denom = denom[:index+1]
-	}
```

Comment:
Why is this removed now?

Reply:
CL denoms no longer contain position data